src/main/java/net/minecraft/world/loot/RandomTagEntry.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2016, 2017, 2018 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.minecraft.world.loot;
```

Comment:
I don't like placing the class into a `net.minecraft` package, but LootChoiceProvider is package-private.

Reply:
I think this can be merged without RandomTagEntry to bypass the problem. Of course, actual usages won't work.